### DISCUSSION TITLE
Support for Whisper's --word_timestamps switch (and maybe other command line options)

### DISCUSSION THREAD

**Discussion by beeeeswax** [SubtitleEdit/subtitleedit#6804] [Ideas]
I've only just realised SE added support for whisper but I've been using whisper stand-alone for quite a while.  I thought it was pretty good but had major problems with timestamp drift, whisper's output would often race ahead by 3-5s before gradually coming back into sync then repeating this cycle.  When the --word_timestamps option was added I tried it immediately and it almost completely solves this problem.

Passing this switch to whisper results in an output that underlines the specific word being spoken at any one time with massively better timestamp accuracy. All I have to do with a --word_timestamps SRT is use "Remove Formatting" to get rid of underlines and then "Merge lines with same text". The results are excellent.

Could this option be made available in SE's implementation of whisper somehow?  Or even better, just expose all the options whisper has at the command line so users can customise their transcription.

Here's an example of the command line I'm using:
`whisper "C:\TVShow\Season 1\TV.Show.S01E01.1080p.HDTV.H.264.DD5.1.ts" --model large-v2 --language en --output_format srt --word_timestamps True --verbose True --output_dir "C:\TVShow\Season 1"`

**Comment by rsmith02ct** [comment]
Interesting idea. Would love to see other tests if they confirm the increased accuracy.
